### ISSUE TITLE
What‘s the training corpus of single GETAR?

### ISSUE THREAD

**Issue by wutianjie** [microsoft/Table-Pretraining#9]
Hi, thanks for your sharing~
Your pretext task designing is very ingenious! But I'm confused about the training corpus' difference between GETAR and GETAR+TAPEX.
In this paper, the accuracy of the single GETAR on TabFact is 80.8%. Is it obtained by **directly fine-tuning BART** on TabFact?  If so, why could BART achieve such competitive performance with [TAPAS](https://arxiv.org/pdf/2010.00571.pdf)(81%, pre-train on 3.7M tables), since BART isn't typical for table understanding?

**Comment by SivilTaram** [comment]
@wutianjie Thanks for your interest on work and your question! 

For the first quesiton, in our most recent arxiv version (https://arxiv.org/pdf/2107.07653.pdf), there is no more term `GETAR`, since it is actually directly using BART on downstream tasks. Therefore, the answer is YES, the performance of 80.8% on TabFact is obtained by **directly fine-tuning BART** on TabFact.

For the second question, I have to say that it is really interesting, but really hard to answer. To be honest, I don't have an answer now either. However, we can observe some trends from our experimental results: by comparing the performance of WikiTableQuestions and TabFact. We can find that with a relatively small amount of data (~10,000) BART does not excel at such a task (TableQA or TableFV); But when there is more data (~100,000) BART can perform the task very well. Therefore, I believe the model itself can learn some table understanding from a rich resource. Therefore, **if you have more than 100,000 examples for a table-related task, you may first try to utilize a pre-trained encoder-decoder model to directly model it, instead of crawling millions of data to perform continual pre-training**.

Another thought I wanna to share is that: we are still not knowing WHY TAPAS-style pre-training objective can work. At least for me, it is still unknown. If TAPAS works well on understanding tables, I think BART can do that in a rich resource scenario. I do not think table understanding will be a bottleneck for table related tasks.

**Comment by SivilTaram** [comment]
@wutianjie Any further discussion is welcome!
